static/src/js/components/DownloadHistory/__tests__/DownloadHistory.test.jsx

```diff
@@ -125,49 +142,28 @@ describe('DownloadHistory component', () => {
         retrievalHistoryLoaded: true,
         onDeleteRetrieval: jest.fn()
       })
-      expect(enzymeWrapper.find(Table).length).toBe(1)
-      expect(enzymeWrapper.find('tbody tr').length).toBe(1)
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('to')).toEqual({
-        pathname: '/downloads/8069076',
-        search: ''
-      })
 
-      expect(enzymeWrapper.find(PortalLinkContainer).prop('children')).toEqual('Collection Title and 1 other collection')
+      expect(screen.getByRole('table')).toBeInTheDocument()
+
+      const link = screen.getByTestId('portal-link')
+      expect(link).toHaveAttribute('href', '/downloads/8069076')
+      expect(link).toHaveTextContent('Collection Title and 1 other collection')
     })
 
-    test('renders the correct Helmet meta information', () => {
-      const { enzymeWrapper } = setup({
-        authToken: 'testToken',
+    test('renders document head metadata correctly', () => {
+      setup({
         earthdataEnvironment: 'prod',
-        retrievalHistory: [{
-          id: '8069076',
-          jsondata: {},
-          created_at: '2019-08-25T11:58:14.390Z',
-          collections: [{
-            title: 'Collection Title'
-          }, {
-            title: 'Collection Title Two'
-          }]
-        }],
+        retrievalHistory: [],
         retrievalHistoryLoading: false,
         retrievalHistoryLoaded: true,
         onDeleteRetrieval: jest.fn()
       })
 
-      const helmet = enzymeWrapper.find(Helmet)
-      expect(helmet.childAt(0).type()).toEqual('title')
-      expect(helmet.childAt(0).text()).toEqual('Download Status & History')
-      expect(helmet.childAt(1).props().name).toEqual('title')
-      expect(helmet.childAt(1).props().content).toEqual('Download Status & History')
-      expect(helmet.childAt(2).props().name).toEqual('robots')
-      expect(helmet.childAt(2).props().content).toEqual('noindex, nofollow')
-      expect(helmet.childAt(3).props().rel).toEqual('canonical')
-      expect(helmet.childAt(3).props().href).toEqual('https://search.earthdata.nasa.gov/downloads')
+      expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent('Download Status & History')
     })
 
     test('renders links correctly when portals were used to place an order', () => {
```

Comment:
None of the logic described in this test is part of DownloadHistory, and should be tested within PortalLinkContainer, this test should be removed.